2018-09-28 17:59:01.458 23159 DEBUG oslo_db.sqlalchemy.engines [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-09-28 17:59:05.383 23159 DEBUG oslo_concurrency.lockutils [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] Lock "d6ebb650-317a-44ff-9b75-bd1a36697143" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-28 17:59:05.387 23159 DEBUG oslo_concurrency.lockutils [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] Lock "d6ebb650-317a-44ff-9b75-bd1a36697143" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-28 17:59:05.415 23159 DEBUG oslo_db.sqlalchemy.engines [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-09-28 17:59:05.629 23159 DEBUG nova.context [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] Found 2 cells: 00000000-0000-0000-0000-000000000000(cell0),d6ebb650-317a-44ff-9b75-bd1a36697143(default) load_cells /usr/lib/python2.7/site-packages/nova/context.py:508
2018-09-28 17:59:05.631 23159 DEBUG oslo_concurrency.lockutils [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-28 17:59:05.632 23159 DEBUG oslo_concurrency.lockutils [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-28 17:59:05.633 23159 DEBUG oslo_concurrency.lockutils [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] Lock "d6ebb650-317a-44ff-9b75-bd1a36697143" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-28 17:59:05.633 23159 DEBUG oslo_concurrency.lockutils [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] Lock "d6ebb650-317a-44ff-9b75-bd1a36697143" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-28 17:59:05.661 23159 DEBUG oslo_db.sqlalchemy.engines [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-09-28 17:59:05.692 23159 DEBUG nova.quota [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] Getting quotas for project 6a9ee05abe2541dd88777a22a0467d60. Resources: set(['instances', 'ram', 'cores']) _get_quotas /usr/lib/python2.7/site-packages/nova/quota.py:442
2018-09-28 17:59:05.701 23159 DEBUG nova.quota [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] Getting quotas for user a8c2104767bb4196b320e2bfe5a4578e and project 6a9ee05abe2541dd88777a22a0467d60. Resources: set(['instances', 'ram', 'cores']) _get_quotas /usr/lib/python2.7/site-packages/nova/quota.py:434
2018-09-28 17:59:05.726 23159 DEBUG oslo_concurrency.lockutils [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] Lock "d6ebb650-317a-44ff-9b75-bd1a36697143" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-28 17:59:05.727 23159 DEBUG oslo_concurrency.lockutils [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] Lock "d6ebb650-317a-44ff-9b75-bd1a36697143" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-28 17:59:05.728 23159 WARNING nova.image.glance [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] No protocol specified in for api_server 'http://10.0.20.23:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2018-09-28 17:59:05.817 23159 DEBUG nova.notifications.objects.base [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] Defaulting the value of the field 'projects' to None in FlavorPayload due to 'Cannot call _load_projects on orphaned Flavor object' populate_schema /usr/lib/python2.7/site-packages/nova/notifications/objects/base.py:125
2018-09-28 17:59:05.829 23159 DEBUG nova.objects.instance [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] Lazy-loading 'tags' on Instance uuid a96eda01-d549-4b5b-b1bf-5a51302766df obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1049
2018-09-28 17:59:05.881 23159 DEBUG nova.conductor.manager [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] [instance: a96eda01-d549-4b5b-b1bf-5a51302766df] block_device_mapping [BlockDeviceMapping(attachment_id=<?>,boot_index=0,connection_info=None,created_at=<?>,delete_on_termination=True,deleted=<?>,deleted_at=<?>,destination_type='local',device_name=None,device_type='disk',disk_bus=None,guest_format=None,id=<?>,image_id='a7c939ef-f3d7-4e38-ba4a-7b518365d38e',instance=<?>,instance_uuid=<?>,no_device=False,snapshot_id=None,source_type='image',tag=None,updated_at=<?>,volume_id=None,volume_size=None)] _create_block_device_mapping /usr/lib/python2.7/site-packages/nova/conductor/manager.py:968
2018-09-28 17:59:05.885 23159 DEBUG oslo_concurrency.lockutils [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] Lock "d6ebb650-317a-44ff-9b75-bd1a36697143" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-28 17:59:05.886 23159 DEBUG oslo_concurrency.lockutils [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] Lock "d6ebb650-317a-44ff-9b75-bd1a36697143" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-28 17:59:05.998 23159 DEBUG oslo_concurrency.lockutils [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] Lock "d6ebb650-317a-44ff-9b75-bd1a36697143" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-28 17:59:05.998 23159 DEBUG oslo_concurrency.lockutils [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] Lock "d6ebb650-317a-44ff-9b75-bd1a36697143" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-28 17:59:06.455 23161 WARNING nova.image.glance [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] No protocol specified in for api_server 'http://10.0.20.23:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2018-09-28 17:59:06.538 23161 DEBUG nova.notifications.objects.base [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] Defaulting the value of the field 'projects' to None in FlavorPayload due to 'Cannot call _load_projects on orphaned Flavor object' populate_schema /usr/lib/python2.7/site-packages/nova/notifications/objects/base.py:125
2018-09-28 17:59:07.006 23166 WARNING nova.image.glance [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] No protocol specified in for api_server 'http://10.0.20.23:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2018-09-28 17:59:07.079 23166 DEBUG nova.notifications.objects.base [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] Defaulting the value of the field 'projects' to None in FlavorPayload due to 'Cannot call _load_projects on orphaned Flavor object' populate_schema /usr/lib/python2.7/site-packages/nova/notifications/objects/base.py:125
2018-09-28 17:59:09.303 23162 WARNING nova.image.glance [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] No protocol specified in for api_server 'http://10.0.20.23:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2018-09-28 17:59:09.375 23162 DEBUG nova.notifications.objects.base [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] Defaulting the value of the field 'projects' to None in FlavorPayload due to 'Cannot call _load_projects on orphaned Flavor object' populate_schema /usr/lib/python2.7/site-packages/nova/notifications/objects/base.py:125
2018-09-28 17:59:09.954 23160 WARNING nova.image.glance [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] No protocol specified in for api_server 'http://10.0.20.23:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2018-09-28 17:59:10.089 23160 DEBUG nova.notifications.objects.base [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] Defaulting the value of the field 'projects' to None in FlavorPayload due to 'Cannot call _load_projects on orphaned Flavor object' populate_schema /usr/lib/python2.7/site-packages/nova/notifications/objects/base.py:125
2018-09-28 17:59:10.659 23167 WARNING nova.image.glance [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] No protocol specified in for api_server 'http://10.0.20.23:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2018-09-28 17:59:10.859 23167 DEBUG nova.notifications.objects.base [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] Defaulting the value of the field 'projects' to None in FlavorPayload due to 'Cannot call _load_projects on orphaned Flavor object' populate_schema /usr/lib/python2.7/site-packages/nova/notifications/objects/base.py:125
2018-09-28 17:59:33.148 23167 WARNING nova.image.glance [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] No protocol specified in for api_server 'http://10.0.20.23:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2018-09-28 17:59:33.171 23167 DEBUG nova.notifications.objects.base [req-a0111d4a-f205-455b-b302-bb4d2cfe454d a8c2104767bb4196b320e2bfe5a4578e 6a9ee05abe2541dd88777a22a0467d60 - a7ad841ad6274572b47170d444357d95 a7ad841ad6274572b47170d444357d95] Defaulting the value of the field 'projects' to None in FlavorPayload due to 'Cannot call _load_projects on orphaned Flavor object' populate_schema /usr/lib/python2.7/site-packages/nova/notifications/objects/base.py:125
